test/fuzz/fuzzer.ml

```diff
@@ -69,6 +74,20 @@ let compare (module I1 : Interprets.INTERPRET)
 
 let check (module I1 : Interprets.INTERPRET) (module I2 : Interprets.INTERPRET)
   m =
+  (* Save the generated module *)
+  if Param.save_modules then (
+    (* Create output directory if it doesn't exist *)
+    if not (Sys.file_exists Param.output_dir) then
+      Unix.mkdir Param.output_dir 0o755;
```

Comment:
Could you use [`Bos.File.Dir.create`](https://github.com/OCamlPro/owi/blob/8100737e7dc19982a5646875ec6a98ed849908d3/src/cmd/cmd_c.ml#L182) instead ? Also, it is better to not tests existence before (the function is going to check it for you).